src/lib/hooks/useTokenList/index.tsx

```diff
@@ -1,113 +1,3 @@
 import { Token } from '@uniswap/sdk-core'
-import { TokenInfo, TokenList } from '@uniswap/token-lists'
-import useActiveWeb3React from 'lib/hooks/useActiveWeb3React'
-import resolveENSContentHash from 'lib/utils/resolveENSContentHash'
-import { createContext, PropsWithChildren, useCallback, useContext, useEffect, useMemo, useState } from 'react'
-import { WrappedTokenInfo } from 'state/lists/wrappedTokenInfo'
-
-import fetchTokenList from './fetchTokenList'
-import { ChainTokenMap, tokensToChainTokenMap } from './utils'
-import { validateTokens } from './validateTokenList'
-
-export { useQueryTokens } from './useQueryTokens'
-
-export const DEFAULT_TOKEN_LIST = 'https://gateway.ipfs.io/ipns/tokens.uniswap.org'
-
-const MISSING_PROVIDER = Symbol()
-const ChainTokenMapContext = createContext<ChainTokenMap | undefined | typeof MISSING_PROVIDER>(MISSING_PROVIDER)
-
-function useChainTokenMapContext() {
-  const chainTokenMap = useContext(ChainTokenMapContext)
-  if (chainTokenMap === MISSING_PROVIDER) {
-    throw new Error('TokenList hooks must be wrapped in a <TokenListProvider>')
-  }
-  return chainTokenMap
-}
-
-export function useIsTokenListLoaded() {
-  return Boolean(useChainTokenMapContext())
-}
-
-export default function useTokenList(): WrappedTokenInfo[] {
-  const { chainId } = useActiveWeb3React()
-  const chainTokenMap = useChainTokenMapContext()
-  const tokenMap = chainId && chainTokenMap?.[chainId]
-  return useMemo(() => {
-    if (!tokenMap) return []
-    return Object.values(tokenMap).map(({ token }) => token)
-  }, [tokenMap])
-}
 
 export type TokenMap = { [address: string]: Token }
```

Comment:
if this is the only thing in this file should we move it?

Reply:
Done.

There are a few other files that have been gutted in a similar way, but I'd prefer to refactor them in a future PR, so that this PR can be isolated to widget code deletion only.

Reply:
no prob, just pointing out the one i noticed. maybe good to list those other things so we have them somewhere, but nbd